sources/clarify.toml

```diff
@@ -168,7 +168,7 @@ license-files = [
 expression = "MIT AND ISC AND OpenSSL"
 license-files = [
     { path = "LICENSE", hash = 0xbd0eed23 },
-    { path = "third_party/fiat/LICENSE", hash = 0xdad0527d },
+    { path = "third_party/fiat/LICENSE", hash = 0x75829ee2 },
```

Comment:
What was the difference in this license?

Reply:
The copyright year was bumped to 2020